bouncer/blacklist/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Create your models here.
+class IP_Entry(models.Model):
+	entry_value = models.GenericIPAddressField()
+
+class Email_Entry(models.Model):
+	entry_value = models.EmailField()
+
+class Host_Entry(models.Model):
```

Comment:
Class names should be `CamelCase`, without `_` underscores.

Also, this makes more sense as `EmailHostEntry`, as we're talking about emails.